compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -29,6 +29,33 @@ namespace luci
 
 namespace
 {
+void compute_sym_scale_zp(float min, float max, float &scaling_factor, int64_t &zp,
+                          float &nudged_min, float &nudged_max)
+{
+  assert(min != max);
+
+  const int32_t kMinScale = std::numeric_limits<int16_t>::lowest() + 1;
+  const int32_t kMaxScale = std::numeric_limits<int16_t>::max();
```

Comment:
Just curiosity, 
1. why does `kMinScale` need `+1`?
2. Maybe `std::numeric_limits<int32_t>` looks right not `int16_t`, doesn't it?

Reply:
1. For symmetric quantization, the range of quantized values should be symmetric (-x ~ x). The lowest() of int16 is -32768 and max() is 32767. So `kMinScale` needs +1.
2. This is for int16 quantization, so `int16_t` would be right.

Reply:
(optional)  maybe
```cxx
const int32_t kMinScale  = -kMaxScale
```

Reply:
I will fix it based on @seanshpark 's comment